README.rst

```diff
@@ -76,22 +76,41 @@ All done! The VM contains all the required dependencies pre-installed.
 Manual
 ^^^^^^
 
+Instructions here have been tested on Fedora 30, using Python3
+
+Get git and python3:
+
+    sudo dnf install git python3
+
 Clone the source::
 
- git clone https://github.com/fedora-infra/nuancier.git
+    git clone https://github.com/fedora-infra/nuancier.git
 
+Install Python requirements
 
+    pip3 install -r requirements.txt
+ 
 Create the database scheme::
 
- python createdb.py
-
+    python3 createdb.py
 
-Run the server::
+Run the server locally::
 
- python runserver.py
+    python3 runserver.py
 
 You should be able to access the server at http://localhost:5000
 
+or to run the server externally, first open relevant port in firewall::
+
+    sudo firewall-cmd --add-port 3000/tcp --permanent
```

Comment:
```suggestion
    sudo firewall-cmd --add-port 5000/tcp --permanent
```

Reply:
You could also forward the port in the Vagrantfile that way you can connect to the application from your host. See https://github.com/fedora-infra/bodhi/blob/develop/Vagrantfile#L53 as an example.

Reply:
Thanks. This looks good. Will also update [example vagrant file](https://github.com/fedora-infra/nuancier/blob/master/Vagrantfile.example) to use a more recent Fedora image and to use Python3